concepts/05 UI Components/NumberBox/00 Getting Started with NumberBox/15 Configure Spin Buttons.md

```diff
@@ -0,0 +1,61 @@
+The NumberBox component allows users to click spin buttons to increase or decrease the input value. To display these buttons, enable the [showSpinButtons](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#showSpinButtons) property. Use the [step](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#step) property to specify a step by which the component value changes when users click the spin buttons, press the Up/Down arrow keys, or roll the mouse wheel. You can also enable the [showClearButton](/Documentation/ApiReference/UI_Components/dxNumberBox/Configuration/#showClearButton) property to show a button that clears the input field.
+
+The following code displays the clear and spin buttons and defines a step the value changes when a user clicks the spin buttons.
```

Comment:
This sentence basically repeats what we said in the previous paragaraph. I think we can safely remove it.

```suggestion
```